.github/workflows/scripts/check_ignore.py

```
@@ -0,0 +1,259 @@
+#!/usr/bin/env python3
+"""Analyze dart files for custom lint rule suppression."""
+
+import argparse
+import os
+import subprocess
+import sys
+
+
+def _arg_parser_resolver():
```

Ruff (0.14.11)

Missing return type annotation for private function `_arg_parser_resolver`

(ANN202)